ansible_collections/arista/avd/molecule/eos_cli_config_gen/documentation/devices/management-security.md

```diff
@@ -90,6 +91,20 @@ interface Management1
 | ------------------ | ----------- | ------ | ------ |
 | - | Hostname must be FQDN | - | Enabled |
 
+### Password Policies
```

Comment:
Could we build this as a table of policies instead of these lists? Like a "repetitive" column with `>6, <20`.